src/pages/alternatives.js

```diff
@@ -161,6 +161,15 @@ const tableData = [
     cloudHosting: true,
     selfHosting: true,
   },
+  {
+    company: "Chiffre",
+    url: "https://chiffre.io",
+    description: "End-to-end encrypted web analytics (hosted in France)",
+    permissiveOpenSource: true,
+    copyleftOpenSource: false,
+    cloudHosting: true,
+    selfHosting: false,
```

Comment:
Technically it can do self-hosted, it's just that the end-to-end encryption overhead makes little sense if you already own your data (might as well self-host something simpler).